tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -629,17 +662,35 @@ platform_tests/test_reboot.py:
     conditions:
       - "hwsku in ['Celestica-E1031-T48S4']"
 
+platform_tests/test_reboot.py::test_fast_reboot:
+  skip:
+    reason: "Skip test_fast_reboot for Nokia-7215"
+    conditions:
+      - "platform in ['armhf-nokia_ixs7215_52x-r0']"
+
 platform_tests/test_reboot.py::test_power_off_reboot:
   skip:
-    reason: "Skip power off reboot test for Wistron"
+    reason: "Skip power off reboot test for Wistron/Nokia-7215"
+    conditions:
+      - "hwsku in ['Celestica-E1031-T48S4'] or 'sw_to3200k' in hwsku or platform in ['armhf-nokia_ixs7215_52x-r0']"
```

Comment:
can you check what is the test intention? why we want to skip it

Reply:
In 7215, we can't get reboot cause